app/database_etl/test_adjustment_handler/test_adjustment.py

```diff
@@ -0,0 +1,246 @@
+                    adj_type = 'Test developer / manufacturer evaluation'
+                se = estimate['sensitivity']
+                sp = estimate['specificity']
+                se_n = estimate['se_n'] if pd.notna(estimate['se_n']) else 30
+                sp_n = estimate['sp_n'] if pd.notna(estimate['sp_n']) else 80
+
+            # Manufacturer evaluation is available
+            elif pd.notna(estimate['sensitivity']) and pd.notna(estimate['specificity']):
+                adj_type = 'Test developer / manufacturer evaluation'
+                se = estimate['sensitivity']
+                sp = estimate['specificity']
+                # per FDA minimum requirements https://www.fda.gov/medical-devices/
+                # coronavirus-disease-2019-covid-19-emergency-use-authorizations-medical-devices/
+                # eua-authorized-serology-test-performance
+                se_n, sp_n = 30, 80
+
+            else:
+                # if there is no matched adjusted estimate available:
+                found_test_type = False
+                for test_type in ['LFIA', 'CLIA', 'ELISA']:
+                    if estimate['test_type'] and test_type == estimate['test_type']:
+                        se = bastos_estimates[test_type]['se']['50']
+                        sp = bastos_estimates[test_type]['sp']['50']
+                        se_n = bastos_estimates[test_type]['se']['n']
+                        sp_n = bastos_estimates[test_type]['sp']['n']
+                        adj_type = 'Used Bastos SR/MA data; no sens, spec, or author adjustment available'
+                        found_test_type = True
+                        break
+                if not found_test_type:
+                    adj_type = 'No data altogether'
+
+            if adj_type == 'No data altogether':
+                adj_prev = None
+                se = None
+                sp = None
+                lower = None
+                upper = None
+            else:
+                print(f'ADJUSTING ESTIMATE AT INDEX {estimate.name}', adj_type)
+                model_params = dict(
+                    n_prev_obs=int(estimate['denominator_value']),
+                    y_prev_obs=int(
+                        estimate['serum_pos_prevalence'] * estimate['denominator_value']),
+                    n_se=int(se_n),
+                    y_se=int(se_n * se),
+                    n_sp=int(sp_n),
+                    y_sp=int(sp_n * sp)
+                )
+                lower, adj_prev, upper = self.pystan_adjust(model_params)
+                if pd.isna(adj_prev):
+                    print(f'FAILED TO ADJUST ESTIMATE AT INDEX {estimate.name}')
+
+        else:
+            adj_type = 'Used author-adjusted estimate'
+            adj_prev = estimate['serum_pos_prevalence']
+            se = estimate['sensitivity']
+            sp = estimate['specificity']
+
+            lower, upper = proportion_confint(int(estimate['denominator_value'] * estimate['serum_pos_prevalence']),
```

Comment:
alpha = 0.05 here please!

Reply:
It would actually be good to map this with the value used for the confidence interval earlier - e.g., if that is 95% then this should be 5%; 90% 10%; etc